models/marts/structure/fct_source_directories.sql

```diff
@@ -6,7 +6,7 @@
 
 with all_graph_resources as (
     select * from {{ ref('int_all_graph_resources') }}
-    where not is_excluded
+    where not cast(is_excluded as {{ dbt.type_boolean() }})
```

Comment:
Do we need to cast this one here in Clickhouse?

Reply:
Actually not, let's remove this